### PR TITLE
Topic messaging

### PR DESCRIPTION
**Pull request by VivaanBahl**

### CODE REVIEW

**Comment by VasuAgrawal** [comment]
did you literally copy / paste from my PR? :P

File: real_time/surface_src/java_src/Tutorial/src/main/java/a_integrator/startercode/IntegerMessage.java [hunk 1]
```diff
@@ -17,6 +17,16 @@ public long getSequenceNumber() {
 	}
 
 	@Override
+	public void setTopicName(String name) {
```
Comment (TrevorDecker) [inline real_time/surface_src/java_src/Tutorial/src/main/java/a_integrator/startercode/IntegerMessage.java]: Codacy Issue found: [Document empty method body](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/2382038834/issues/source?bid=2938665&fileBranchId=3226080#l20)

File: real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/robots/TransistorPlayback.java [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+import com.roboclub.robobuggy.nodes.sensors.RBSMNode;
+import com.roboclub.robobuggy.ros.NodeChannel;
+import com.roboclub.robobuggy.simulation.SensorPlayer;
+import com.roboclub.robobuggy.ui.ConfigurationPanel;
+import com.roboclub.robobuggy.ui.Gui;
+import com.roboclub.robobuggy.ui.ImuPanel;
+import com.roboclub.robobuggy.ui.ImuVisualWindow;
+import com.roboclub.robobuggy.ui.MainGuiWindow;
+import com.roboclub.robobuggy.ui.PathPanel;
+import com.roboclub.robobuggy.ui.PoseGraphsPanel;
```
Comment (TrevorDecker) [inline real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/robots/TransistorPlayback.java]: Codacy Issue found: [Avoid unused imports such as 'com.roboclub.robobuggy.ui.PoseGraphsPanel'](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/2382038752/issues/source?bid=2938665&fileBranchId=3226080#l20)

File: real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/robots/TransistorPlayback.java [hunk 3]
```diff
@@ -0,0 +1,67 @@
+package com.roboclub.robobuggy.robots;
+
+import com.roboclub.robobuggy.main.RobobuggyConfigFile;
+import com.roboclub.robobuggy.main.RobobuggyLogicNotification;
+import com.roboclub.robobuggy.main.RobobuggyMessageLevel;
+import com.roboclub.robobuggy.nodes.localizers.HighTrustGPSLocalizer;
+import com.roboclub.robobuggy.nodes.sensors.CameraNode;
```
Comment (TrevorDecker) [inline real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/robots/TransistorPlayback.java]: Codacy Issue found: [Avoid unused imports such as 'com.roboclub.robobuggy.nodes.sensors.CameraNode'](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/2382038752/issues/source?bid=2938665&fileBranchId=3226080#l7)

File: real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/robots/TransistorPlayback.java [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+import com.roboclub.robobuggy.nodes.sensors.GpsNode;
+import com.roboclub.robobuggy.nodes.sensors.HillCrestImuNode;
```
Comment (TrevorDecker) [inline real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/robots/TransistorPlayback.java]: Codacy Issue found: [Avoid unused imports such as 'com.roboclub.robobuggy.nodes.sensors.HillCrestImuNode'](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/2382038752/issues/source?bid=2938665&fileBranchId=3226080#l9)

File: real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/robots/TransistorPlayback.java [hunk 5]
```diff
@@ -0,0 +1,67 @@
+package com.roboclub.robobuggy.robots;
+
+import com.roboclub.robobuggy.main.RobobuggyConfigFile;
+import com.roboclub.robobuggy.main.RobobuggyLogicNotification;
+import com.roboclub.robobuggy.main.RobobuggyMessageLevel;
+import com.roboclub.robobuggy.nodes.localizers.HighTrustGPSLocalizer;
```
Comment (TrevorDecker) [inline real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/robots/TransistorPlayback.java]: Codacy Issue found: [Avoid unused imports such as 'com.roboclub.robobuggy.nodes.localizers.HighTrustGPSLocalizer'](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/2382038752/issues/source?bid=2938665&fileBranchId=3226080#l6)

File: real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/robots/TransistorPlayback.java [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+import com.roboclub.robobuggy.nodes.sensors.GpsNode;
+import com.roboclub.robobuggy.nodes.sensors.HillCrestImuNode;
+import com.roboclub.robobuggy.nodes.sensors.LoggingNode;
+import com.roboclub.robobuggy.nodes.sensors.RBSMNode;
+import com.roboclub.robobuggy.ros.NodeChannel;
+import com.roboclub.robobuggy.simulation.SensorPlayer;
+import com.roboclub.robobuggy.ui.ConfigurationPanel;
+import com.roboclub.robobuggy.ui.Gui;
+import com.roboclub.robobuggy.ui.ImuPanel;
+import com.roboclub.robobuggy.ui.ImuVisualWindow;
```
Comment (TrevorDecker) [inline real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/robots/TransistorPlayback.java]: Codacy Issue found: [Avoid unused imports such as 'com.roboclub.robobuggy.ui.ImuVisualWindow'](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/2382038752/issues/source?bid=2938665&fileBranchId=3226080#l17)

File: real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/robots/TransistorPlayback.java [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+import com.roboclub.robobuggy.nodes.sensors.CameraNode;
+import com.roboclub.robobuggy.nodes.sensors.GpsNode;
+import com.roboclub.robobuggy.nodes.sensors.HillCrestImuNode;
+import com.roboclub.robobuggy.nodes.sensors.LoggingNode;
+import com.roboclub.robobuggy.nodes.sensors.RBSMNode;
+import com.roboclub.robobuggy.ros.NodeChannel;
+import com.roboclub.robobuggy.simulation.SensorPlayer;
+import com.roboclub.robobuggy.ui.ConfigurationPanel;
+import com.roboclub.robobuggy.ui.Gui;
+import com.roboclub.robobuggy.ui.ImuPanel;
```
Comment (TrevorDecker) [inline real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/robots/TransistorPlayback.java]: Codacy Issue found: [Avoid unused imports such as 'com.roboclub.robobuggy.ui.ImuPanel'](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/2382038752/issues/source?bid=2938665&fileBranchId=3226080#l16)

File: real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/robots/TransistorPlayback.java [hunk 8]
```diff
@@ -0,0 +1,67 @@
+package com.roboclub.robobuggy.robots;
+
+import com.roboclub.robobuggy.main.RobobuggyConfigFile;
+import com.roboclub.robobuggy.main.RobobuggyLogicNotification;
+import com.roboclub.robobuggy.main.RobobuggyMessageLevel;
+import com.roboclub.robobuggy.nodes.localizers.HighTrustGPSLocalizer;
+import com.roboclub.robobuggy.nodes.sensors.CameraNode;
+import com.roboclub.robobuggy.nodes.sensors.GpsNode;
```
Comment (TrevorDecker) [inline real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/robots/TransistorPlayback.java]: Codacy Issue found: [Avoid unused imports such as 'com.roboclub.robobuggy.nodes.sensors.GpsNode'](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/2382038752/issues/source?bid=2938665&fileBranchId=3226080#l8)

File: real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/robots/TransistorPlayback.java [hunk 9]
```diff
@@ -0,0 +1,67 @@
...
+import com.roboclub.robobuggy.nodes.sensors.HillCrestImuNode;
+import com.roboclub.robobuggy.nodes.sensors.LoggingNode;
+import com.roboclub.robobuggy.nodes.sensors.RBSMNode;
```
Comment (TrevorDecker) [inline real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/robots/TransistorPlayback.java]: Codacy Issue found: [Avoid unused imports such as 'com.roboclub.robobuggy.nodes.sensors.RBSMNode'](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/2382038752/issues/source?bid=2938665&fileBranchId=3226080#l11)

File: real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/robots/TransistorPlayback.java [hunk 10]
```diff
@@ -0,0 +1,67 @@
...
+import com.roboclub.robobuggy.ros.NodeChannel;
+import com.roboclub.robobuggy.simulation.SensorPlayer;
+import com.roboclub.robobuggy.ui.ConfigurationPanel;
+import com.roboclub.robobuggy.ui.Gui;
+import com.roboclub.robobuggy.ui.ImuPanel;
+import com.roboclub.robobuggy.ui.ImuVisualWindow;
+import com.roboclub.robobuggy.ui.MainGuiWindow;
+import com.roboclub.robobuggy.ui.PathPanel;
```
Comment (TrevorDecker) [inline real_time/surface_src/java_src/Alice/src/main/java/com/roboclub/robobuggy/robots/TransistorPlayback.java]: Codacy Issue found: [Avoid unused imports such as 'com.roboclub.robobuggy.ui.PathPanel'](https://www.codacy.com/app/Mr-Trevor-Decker/RoboBuggy/file/2382038752/issues/source?bid=2938665&fileBranchId=3226080#l19)

**Comment by VivaanBahl** [comment]
this was taken care of by Vasu

**Pull request closed without merging**